app/Http/Controllers/Legacy/Web/CampaignIdsController.php

```diff
@@ -15,8 +15,7 @@ class CampaignIdsController extends Controller
     public function __construct()
     {
         $this->middleware('auth');
-        $this->middleware('role:admin,staff', ['only' => ['index', 'show']]);
-        $this->middleware('role:admin', ['only' => ['create', 'store', 'edit', 'update', 'destroy']]);
```

Comment:
I removed this because I think we want staff to also be able to do all of this, not sure why before it only allowed admins to do this.

Reply:
@chloealee I think at one time we had planned to only let admins touch the campaigns stuff...that sounds familiar to me at least

Reply:
@katiecrane ah yes you're totally right, thank you for jogging my memory on that!